### PR TITLE
Uior 242 add metadata widget to the top of the po and pol record

### PR DESCRIPTION
**Pull request by Feboll**
## Purpose

See metadata widget.
https://issues.folio.org/browse/UIOR-242

## Approach

Added metadata widget to PO and POL details view.

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/components/PurchaseOrder/PODetails/PODetailsView.js [hunk 1]
```diff
@@ -26,92 +27,107 @@ class PODetailsView extends Component {
     const addressBillTo = get(addresses.find(el => el.id === get(order, 'billTo', '')), 'address', '');
     const addressShipTo = get(addresses.find(el => el.id === get(order, 'shipTo', '')), 'address', '');
 
+    const metadata = get(order, 'metadata');
```
Comment (aliaksei-chumakou) [inline src/components/PurchaseOrder/PODetails/PODetailsView.js]: I think you need to use something like this
`{initialValues.metadata && <this.cViewMetaData metadata={initialValues.metadata} />}`
https://github.com/folio-org/ui-invoice/pull/17/files#diff-08aec453035a9e78f35e04a26b8a7280R115

Comment (NikitaSedyx) [reply src/components/PurchaseOrder/PODetails/PODetailsView.js]: and you need `ViewMetaData`

Comment (NikitaSedyx) [reply src/components/PurchaseOrder/PODetails/PODetailsView.js]: and you need `ViewMetaData`

Comment (Feboll) [reply src/components/PurchaseOrder/PODetails/PODetailsView.js]: Done

File: src/components/POLine/POLineView.js [hunk 2]
```diff
@@ -289,6 +292,15 @@ class POLineView extends Component {
             label={<FormattedMessage id="ui-orders.line.accordion.itemDetails" />}
             id="ItemDetails"
           >
+            {metadata && <ViewMetaData metadata={metadata} />}
+            {false && <MetaSection
```
Comment (NikitaSedyx) [inline src/components/POLine/POLineView.js]: mm, you forgot to remove this

File: src/components/PurchaseOrder/PODetails/PODetailsView.js [hunk 3]
```diff
@@ -26,92 +27,101 @@ class PODetailsView extends Component {
...
-            label={<FormattedMessage id="ui-orders.poLine.owner" />}
-            value={get(order, 'owner')}
-          />
-        </Col>
-        <Col xs={12}>
-          {get(order, 'notes', []).map((note, index) => (
+      <Fragment>
+        <Row>
+          <Col xs={12}>
+            {metadata && <ViewMetaData metadata={metadata} />}
```
Comment (NikitaSedyx) [inline src/components/PurchaseOrder/PODetails/PODetailsView.js]: just curious, does it work without stripes connect?

Comment (Feboll) [reply src/components/PurchaseOrder/PODetails/PODetailsView.js]: Yes, it work)

Comment (aliaksei-chumakou) [reply src/components/PurchaseOrder/PODetails/PODetailsView.js]: could you please check  if it works on testing? thanks

**Pull request merged**